### ISSUE TITLE
Weather forecasting data

### ISSUE THREAD

**Issue by rcrowder** [nupic-community/river-view#93]
Just read http://www.randalolson.com/2015/08/13/the-new-york-times-weather-chart-redux/ that is scrapping from http://www.wunderground.com/
Useful and appropriate?

**Comment by rhyolight** [comment]
Certainly, especially the [scraper](https://github.com/rhiever/Data-Analysis-and-Machine-Learning-Projects/blob/master/us-weather-history/wunderground_scraper.py) and [parser](https://github.com/rhiever/Data-Analysis-and-Machine-Learning-Projects/blob/master/us-weather-history/wunderground_parser.py).
